app/main.cpp

```diff
@@ -235,6 +237,7 @@ int main( int argc, char *argv[] )
   // Set/Get enviroment
   QString dataDir = getDataDir( IS_TEST );
   QString projectDir = dataDir + "/projects/";
+  LOG_FILE = projectDir + ".logs";
```

Comment:
Would be better to have the log filename defined in InputUtils - and have a function to set the log filename path... e.g. InputUtils::setLogFilename( ... )